support-frontend/assets/pages/paper-subscription-landing/components/content/paperProductInfo.tsx

```diff
@@ -0,0 +1,114 @@
+import { css } from '@emotion/react';
```

Comment:
A lot of this file is copied from hero.tsx, right? Is it necessary to copy it?

Reply:
This component has been based upon 'productinfo.tsx' and then modified. Think re-using one depends how far modified it is, whether it will be reused in future and how simple/readable it will be. In this case it's an extra prop, change of css and different copy functions, do you think worth it?

Reply:
Nah, I think it’s probably fine – I guess copying like this will make it easier to remove once the test is finished

Reply:
Great thanks